src/components/back-to-top/back-to-top.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ArrowUp from "../../images/icons/circle-up.svg";
+import "./back-to-top.scss";
+const BackToTop = ({ width, height }) => {
+  return (
+    <div
+      className="back-to-top"
+      onClick={window.scrollTo({

```

Comment:
Is suport for IE needed? `scrollTo` does not work on IE11, I suggest using `scrollTop = 0`.

scrollTop needs to be added on the element with scroll, if the document has scroll then you can use `window.document.documentElement.scrollTop = 0`

Reply:
I was mistaken :), scrollTo works on ie11